schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaLoader.java

```diff
@@ -67,10 +67,14 @@ static int mainInternal(String... args) {
 
   @Nullable
   private static Object getCommand(String arg) {
+    if (arg.equals("--config")
+        || arg.startsWith("--config=")
+        || arg.equals("-c")
+        || arg.startsWith("-c=")) {
```

Comment:
In addition to `--config` and `-c`, We should also allow arguments starting with `--config=` and `-c=`.